### PR TITLE
Updated and corrected German lang file

### PR DESCRIPTION
**Pull request by dgelessus**
A few achievements are not fully translated and are commented out. Suggestions and bad puns are welcome.

> `Erlaubt es, Informationen über Blöcke anzuzeigen, wie zum Bleistift ihre §fAdresse§7` ...

Autocorrect FTW.

### CODE REVIEW

File: src/main/resources/assets/opencomputers/lang/de_DE.lang [hunk 1]
```diff
@@ -233,18 +242,19 @@ oc:tooltip.AbstractBusCard=Erlaubt es, LIP-Pakete des Abstrakten Busses von §fS
 oc:tooltip.Acid=Eine hochgiftige Möchtegernflüssigkeit, wird üblicherweise nur von gewissen Piraten konsumiert. Dank ihrer korrosiven Eigenschaften ideal zum Bedrucken von Leiterplatten geeignet.
 oc:tooltip.Adapter=Erlaubt es, Blöcke anzusteuern, die keine Komponentenblöcke sind, wie etwa reguläre Minecraft-Blöcke oder Blöcke anderer Mods.
 oc:tooltip.ALU=Zählt Zahlen zum Zeitvertreib. Klingt komisch, is aber so.
-oc:tooltip.Analyzer=Erlaubt es, Informationen über Blöcke anzuzeigen, wie zum Bleistift ihre §fAdresse§7 und ihren §fKomponentennamen§7.[nl] Erlaubt zudem, den Fehler anzuzeigen, der zu einem Computerabsturz geführt hat, falls der Computer nicht regulär heruntergefahren wurde.
+oc:tooltip.Analyzer=Erlaubt es, Informationen über Blöcke anzuzeigen, wie zum Beispiel ihre §fAdresse§7 und ihren §fKomponentennamen§7.[nl] Erlaubt zudem, den Fehler anzuzeigen, der zu einem Computerabsturz geführt hat, falls der Computer nicht regulär heruntergefahren wurde.
```
Comment (Vexatos) [inline src/main/resources/assets/opencomputers/lang/de_DE.lang]: `Bleistift` is very much ~~indented~~intended here. Please revert this change.

File: src/main/resources/assets/opencomputers/lang/de_DE.lang [hunk 2]
```diff
@@ -183,17 +192,17 @@ oc:gui.Error.NoCPU=Im Computer ist kein Hauptprozessor (CPU) installiert.
...
+oc:gui.Manual.Blocks=OpenComputers-Blöcke
 oc:gui.Manual.Home=Startseite
-oc:gui.Manual.Items=OpenComputers Gegenstände
+oc:gui.Manual.Items=OpenComputers-Gegenstände
 oc:gui.Manual.Warning.BlockMissing=Block nicht verfügbar.
 oc:gui.Manual.Warning.ImageMissing=Bild nicht gefunden.
 oc:gui.Manual.Warning.ItemMissing=Gegenstand nicht verfügbar.
 oc:gui.Manual.Warning.OreDictMissing=Ore-Dictionary-Eintrag nicht verfügbar.
-oc:gui.Raid.Warning=§4Eine Platte einzufügen löscht sie.[nl] Eine Platte zu entfernen löscht das Raid.
+oc:gui.Raid.Warning=§4Festplatten werden beim Einsetzen gelöscht.[nl] Beim Entfernen einer Platte wird das gesamte Raid gelöscht.
```
Comment (fnuecke) [inline src/main/resources/assets/opencomputers/lang/de_DE.lang]: Did you check if this actually still fits into the GUI? IIRC the admittedly sub-par wording was due to space constraints.

File: src/main/resources/assets/opencomputers/lang/de_DE.lang (hunk 1 above)
Comment (fnuecke) [inline src/main/resources/assets/opencomputers/lang/de_DE.lang]: Yep, intentional indeed. Don't ruin my bad jokes! :P

Comment (dgelessus) [inline src/main/resources/assets/opencomputers/lang/de_DE.lang]: My bad joke understanding skills are apparently not good enough. Yet. Does the Bleistift thing only refer to bad autocorrect, or because the analyzer can display errors, or because you might write your name and address with one, or whaa? I'm confused...

File: src/main/resources/assets/opencomputers/lang/de_DE.lang (hunk 2 above)
Comment (dgelessus) [inline src/main/resources/assets/opencomputers/lang/de_DE.lang]: Right, that text is displayed in the raid GUI, forgot about that. Even if there are space issues, Festplatte could be shortened to Platte, and the second part could be written as "Entfernen einer Platte löscht das Raid."

File: src/main/resources/assets/opencomputers/lang/de_DE.lang (hunk 1 above)
Comment (fnuecke) [inline src/main/resources/assets/opencomputers/lang/de_DE.lang]: Google it :P It's just a comedic bastardization of "zum Beispiel", because it sounds somewhat similar. Like "Kentucky schreit *cken". I'd thought it was pretty common, but maybe I'm getting old >_>

Comment (dgelessus) [inline src/main/resources/assets/opencomputers/lang/de_DE.lang]: Hm. I found a very informative article on the subject on sueddeutsche.de. I am confused, astonished and disappointed at the same time. Maybe I'll let you keep this one, if I remember tomorrow :P

**Comment by dgelessus** [comment]
The Raid warning is now three lines long, but sounds more "normal" than before. Could you move the text up by two pixels so the third line doesn't clip the inventory heading?

File: src/main/resources/assets/opencomputers/lang/de_DE.lang [hunk 3]
```diff
@@ -117,12 +123,14 @@ item.oc.UpgradeContainerCard2.name=Karten-Behälter (Stufe 3)
 item.oc.UpgradeContainerUpgrade0.name=Upgrade-Behälter (Stufe 1)
 item.oc.UpgradeContainerUpgrade1.name=Upgrade-Behälter (Stufe 2)
 item.oc.UpgradeContainerUpgrade2.name=Upgrade-Behälter (Stufe 3)
-item.oc.UpgradeCrafting.name=Fertigungs-Upgrade
+item.oc.UpgradeCrafting.name=Crafting-Upgrade
```
Comment (Vexatos) [inline src/main/resources/assets/opencomputers/lang/de_DE.lang:127]: General convention sais that Crafting should be translated as "Fertigung" in automation environments (and robots are a kind of Automation).

Comment (dgelessus) [inline src/main/resources/assets/opencomputers/lang/de_DE.lang:127]: Personally I found the wording a bit unclear. In all other cases you have some context and can guess that Fertigung refers to combining items in a crafting grid, but here there's nothing else to indicate what exactly this upgrade does (well, except for the icon). I found Crafting easier to understand here than Fertigung, but maybe that's just me.

**Comment by dgelessus** [comment]
Translation file should be complete, let me know if anything needs to be fixed.

**Comment by fnuecke** [comment]
OK. I'll have a look at the raid's GUI when I can. Oh, and squash it, please, thanks!
